packages/wallet/src/components/Utils/Drawer.tsx

```diff
@@ -53,14 +52,6 @@ export const CustomDrawer = ({ id, computer }: any) => {
     initFlowbite()
   }, [])
 
-  const refreshBalance = useCallback(async () => {
```

Comment:
I'm not sure if we might need this block

Reply:
have added it and kept it as a comment